components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -966,16 +1052,15 @@ public List<String> getOIDCAudiences(String tenantDomain, String consumerKey) th
                 }
             }
         } catch (SQLException e) {
-            String errorMsg = "Error occurred while retrieving OIDC audiences for client ID: " + consumerKey +
-                    " and tenant domain: " + tenantDomain;
+            String errorMsg = "Error occurred while retrieving OIDC Access Token audiences for client ID: " +
+                    consumerKey + " and tenant domain: " + tenantDomain;
             IdentityDatabaseUtil.rollbackTransaction(connection);
             throw new IdentityOAuth2Exception(errorMsg, e);
         } finally {
             IdentityDatabaseUtil.closeAllConnections(connection, rSetAudiences, prepStmt);
         }
         return audiences;
     }
-
```

Comment:
add this line back

Reply:
fixed with 95271dc3c7cc820ee8b77a92ce75c0129db0015b